README.md

```diff
@@ -87,27 +103,21 @@ projects:
     version: '7.50'
 ```
 
-In _/src/make/drupal.make_ we can also define the contributed modules, themes, and libraries that our site uses. For example if our site uses the deploy module we can add this to _/src/make/drupal.make_ under the `projects` section:
+In _/src/make/drupal.make_ we can also define the contributed modules, themes, and libraries that our site uses. For example if our site uses the [Deploy](https://www.drupal.org/project/deploy) module we can add this to _/src/make/drupal.make_ under the `projects` section:
 
 ```yaml
 projects:
   deploy:
     version: '3.1'
 ```
 
```

Comment:
If our site required a custom patch to the deploy module, we add a _src/patches/custom_patch.patch_ directory. For public patches we just need the url to the patch.
```yaml
projects:
    deploy:
        version: '3.1'
        patch:
            1: '../patches/custom_patch.patch
            3005415: 'https://www.drupal.org/files/issues/2018-10-09/use_plain_text_format-3005415.patch'
```